src/ContaoManager/Plugin.php

```diff
@@ -31,28 +31,19 @@
 
 class Plugin implements BundlePluginInterface, RoutingPluginInterface, ExtensionPluginInterface
 {
-    private $arrLoadAfter = [
-        ContaoCoreBundle::class,
-        ContaoCalendarBundle::class,
-        ContaoFaqBundle::class,
-        ContaoNewsBundle::class
-    ];
-
     /**
      * {@inheritdoc}
      */
     public function getBundles(ParserInterface $parser)
     {
-        $loadAfter = [];
-        foreach ($this->arrLoadAfter as $className) {
-            if (class_exists($className)) {
-                $loadAfter[] = $className;
-            }
-        }
-
         return [
             BundleConfig::create(SiowebGlossarBundle::class)
-                ->setLoadAfter($loadAfter)
+                ->setLoadAfter([
+                    ContaoCoreBundle::class,
+                    ContaoCalendarBundle::class,
+                    ContaoFaqBundle::class,
+                    ContaoNewsBundle::class
+                ])
```

Comment:
Note on this change: you do not have to actually check for the existence of the bundle yourself here. The bundle config will ignore any classes that do not exist anyway.